src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newObject = {};
+
+  for (const [key, value] of Object.entries(robot)) {
+    if (newObject[value] === undefined) {
```

Comment:
Instead of checking for undefined with 'newObject[value] === undefined', consider using 'newObject.hasOwnProperty(value)' which is a more reliable way to check if a property exists in an object.